package.json

```diff
@@ -16,10 +16,10 @@
     "//comment1": "NB: Dynamic index html generation (SSO)",
     "//comment2": "Before deploy, do prod build, and move index.html to functions folder",
     "deploy": "mv dist/index.html functions/ && firebase deploy --only functions,hosting",
-    "e2e:serve_and_retest_on_change": "xvfb-maybe start-server-and-test serve:dev http://localhost:4200 'npm run e2e:retest_on_change'",
-    "e2e:retest_on_change": "xvfb-maybe nodemon -e js,jsx,ts,tsx,css,scss --exec 'ng test -- --watch=false; cypress run --headed --no-exit'",
-    "e2e:autorun": "xvfb-maybe ng test -- --watch=false; xvfb-maybe cypress run --headed",
-    "e2e:serve_and_test": "npx cypress install; xvfb-maybe start-server-and-test serve:dev http://localhost:4200 'ng test -- --watch=false; cypress run'",
+    "test:serve_and_retest_on_change": "xvfb-maybe start-server-and-test serve:dev http://localhost:4200 'npm run test:retest_on_change'",
+    "test:retest_on_change": "xvfb-maybe nodemon -e js,jsx,ts,tsx,css,scss --exec 'ng test --watch=false; cypress open --config watchForFileChanges=true'",
+    "test:autorun": "xvfb-maybe ng test --watch=false; xvfb-maybe cypress run --headed",
+    "Compodoc": "npx cypress install; xvfb-maybe start-server-and-test serve:dev http://localhost:4200 'ng test --watch=false; cypress run'",
```

Comment:
What's Compodoc?

Reply:
:upside_down_face: This is a really good question @lukestanley! I have no idea :D 
Some time as passed and I don't know why I put that name. Well, obviously is not a good name :sweat_smile:

Reply:
Clearly, it has little to do with Compodoc and should be fixed! @gsambrotta